storage/remote/intern_test.go

```diff
@@ -36,52 +37,55 @@ func TestIntern(t *testing.T) {
 }
 
 func TestIntern_MultiRef(t *testing.T) {
+	mgr := &QueueManager{interner: newPool()}
```

Comment:
you can just do `interner := newPool()` in each test

Reply:
do you mean interner = newPool()? Should'nt we avoid global variables?

Reply:
No, `interner = newPool()` would be reusing an existing var named `interner` from somewhere. The point is that we don't need to create an instance of the `QueueManager` struct in these tests.

Reply:
Done!